src/components/Leaderboard.tsx

```diff
@@ -75,7 +19,7 @@ export interface IProps {
 }
 
 const Leaderboard: React.FC<IProps> = ({ content, limit = 100 }) => (
-  <Table>
+  <Box bgColor="background" boxShadow="tableBoxShadow" w="100%" mb={8}>
```

Comment:
`tableBoxShadow` doesn't exist in the Chakra theme. You will need to create it under https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/foundations/shadows.ts
Please, remove "Shadow" sufix when you do it. Use the name `tableBox`.